fleetshard/pkg/testutils/k8s.go

```diff
@@ -72,14 +72,17 @@ func (t reconcileTracker) Create(gvr schema.GroupVersionResource, obj runtime.Ob
 		route.Status = admittedStatus()
 	}
 	if err := t.ObjectTracker.Create(gvr, obj, ns); err != nil {
-		return err
+		return fmt.Errorf("creating group version resource: %w", err)
 	}
 	if gvr == centralsGVR {
 		var multiErr *multierror.Error
 		multiErr = multierror.Append(multiErr, t.ObjectTracker.Create(secretsGVR, newCentralTLSSecret(ns), ns))
```

Comment:
I wasn't quite sure how to tackle this one, but in order to not loose context I was wondering if it would be better to not throw the `err` from the `Create` calls directly into the `multiErr`, but instead add wrapped errors (providing context as in l.75 above) to the multiErr.
WDYT?

Reply:
Could have something like this, WDYT?

```suggestion
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(secretsGVR, newCentralTLSSecret(ns), ns), "creating central TLS secret"))
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(routesGVR, newCentralRoute(ns), ns), "creating central route"))
		multiErr = multierror.Append(multiErr, errors.Wrap(t.ObjectTracker.Create(routesGVR, newCentralMtlsRoute(ns), ns), "creating central MTLS route"))		
```